drivers/serial/Kconfig.nrfx

```diff
@@ -57,6 +57,16 @@ config UART_ASYNC_TX_CACHE_SIZE
 	  in RAM, because EasyDMA in UARTE peripherals can only transfer data
 	  from RAM.
 
+config UART_NRFX_UARTE_RX_FLUSH_MAGIC_BYTE
+	int "Byte used for RX FIFO flush workaround"
+	default 171
```

Comment:
Was it intentional that now the default magic byte is different (0xAB) than previously used 0xAA?
And wouldn't a hexadecimal number be more natural here?

Reply:
Yes, it was intentional, as BT guys asked to use something different than 0xAA because their tests were hitting an error due to 0xAA use. However, today I've got information from them that 0xAB is also not good. We will be looking for another solution to their issue so I get back here to 0xAA. I was not aware of `hex`. Changed to use that.